### ISSUE TITLE
Adjust Biosilicified Chitin Boots so they have 100% coverage

### ISSUE THREAD

**Issue by Jan-Blasiak** [CleverRaven/Cataclysm-DDA#74333]
### Is your feature request related to a problem? Please describe.

I've discovered that Biosilicified Chitin Boots do not protect against acid, despite the fact that the acid protection is basically the only good thing about Biosilicified Chitin.

### Solution you would like.

I would like to see the coverage increased to 100%, so the boots made out of acidproof material actually protect from acid.

### Describe alternatives you have considered.

Acidproof socks?

The recipe for additionally lining the boots with rubber?

The option to add the chitin around rubber boots/normal boots to make **those** acidproof?

### Additional context

Biosilicified Chitin is already a bad armor material. Armor made from it has almost no protection against bash damage, which is THE most common type of damage in the game. Artificially nerfing the boots by making holes in them completely nullifies the reason for it's existence.

TLDR: Green ant armor is cool, make it also useful.

(_it's literally one line edit, grumble grumble, I could do it myself if I could get an approval_)

**Comment by PatrikLundell** [comment]
I agree that acid ant chitin is a big noob trap currently.

**Comment by Jan-Blasiak** [comment]
I've edited the boots in my own game and the difference finally makes these boots worthwhile.

They are still shit armor wise, but the utility makes them very much worth it.

**ALSO** I stopped getting warnings about stepping on prickly stuff which was incredibly annoying.

100% coverage NAO

**Comment by zachary-kaelan** [comment]
You can see in #71100 that making the boots completely acidproof was rejected.

**Comment by Maleclypse** [comment]
So I think the previous issue has been that you wouldn't have the chitin on the soles of the feet, but if you were salvaging rubber soles from another shoe and combining that with the chitin it would make sense to have 100% coverage.  I agree it's a stupid recipe that doesn't make sense to use currently.  I wonder if we have restaurant shoes that have the rubber soles that make them no slip.  So you could up armor those shoes and that would make some sense to now have 100% acid coverage?
